app/controllers/users/piv_cac_setup_from_sign_in_controller.rb

```diff
@@ -10,8 +10,6 @@ class PivCacSetupFromSignInController < ApplicationController
     def prompt
       if params.key?(:token)
         process_piv_cac_setup
-      elsif flash[:error_type].present?
```

Comment:
this controller was never able to reach these error branches, so I removed them.